cpp/src/tutorial1_display.cpp

```diff
@@ -88,32 +54,31 @@ int video(int width, int height, std::string pixel_format, int num_device){
     for (int i = 0; i < num_device; ++i){
       output.push_back(Halide::Buffer<T>(buf_size));
     }
-    pm.set(n["output"], output);
+    n["output"].bind(output);
 
-    int loop_num = 1000;
+    int loop_num = 100;
+    int coef =  positive_pow(2, num_bit_shift_map[pixel_format]);
     for (int i = 0; i < loop_num; ++i)
     {
-      pm.set(dispose_p, i == loop_num-1);
       // JIT compilation and execution of pipelines with Builder.
       try {
-          b.run(pm);
+        b.run();
       }catch(std::exception& e){
```

Comment:
We can delete try-catch from here and delegate to outermost try-catch block.

Reply:
fixed in  a85a0a5